### ISSUE TITLE
Terraform fmt does not convert EOL after comment.

### ISSUE THREAD

**Issue by agarstang** [hashicorp/terraform#13845]
Hi there,

`terraform fmt` does not convert EOL after comment when converting from `<CR><LF>` to `<LF>`.

### Terraform Version
`0.9.3`

### Terraform Configuration Files
```hcl
variable "test1" {
  default = "adam1"
}
#Test Line Comment
variable "test2" {
  default = "adam2" # Test comment
}

variable "test3" {
  default = "adam3"
}
```

### Actual Behavior
Running `terraform fmt` on the above configuration terminated with `<CR><LF>` produces the below output.
```hcl
variable "test1" {<LF>
  default = "adam1"<LF>
}<LF>
#Test Line Comment<CR><LF>
variable "test2" {<LF>
  default = "adam2" # Test Inline Comment<CR><LF>
}<LF>
<LF>
variable "test3" {<LF>
  default = "adam3"<LF>
}<LF>
```

### Expected Behavior
EOL after comment should not be ignored. All EOLs should be changed to `<LF>`.

Regards,
Adam

**Comment by jbardin** [comment]
Hi,

Thanks for filing this issue. 
There's an upstream PR pending for HCL which will take care of this: https://github.com/hashicorp/hcl/pull/173

**Comment by jbardin** [comment]
Fixed via #14225
